Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Registration.hs

```diff
@@ -52,14 +53,16 @@ data LoginReq = LoginReq
   { email :: Maybe Text,
     password :: Text,
     merchantId :: ShortId DMerchant.Merchant,
-    otp :: Maybe Text
+    otp :: Maybe Text,
+    city :: City.City
```

Comment:
Not backward compatible. It must be Maybe.

Reply:
Fixed. Please check.